From e88ff4d294af995c671760e2a0741a646a52edca Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 20 Dec 2019 00:34:16 +0300
Subject: [PATCH 2/2] arm64: dts: renesas: salvator-common: Add "tee" MTD RPC
 HyperFlash partition

This reduces "bl31" partition size, and inserts "tee" partition.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/salvator-common.dtsi | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index 6215545..5083d92 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -848,7 +848,11 @@
 				read-only;
 			};
 			bl31@001C0000 {
-				reg = <0x001C0000 0x480000>;
+				reg = <0x001C0000 0x040000>;
+				read-only;
+			};
+			tee@00200000 {
+				reg = <0x00200000 0x440000>;
 				read-only;
 			};
 			uboot@00640000 {
-- 
2.7.4

